tests/runtests.pl

```diff
@@ -140,6 +140,9 @@ BEGIN
 my %disabled_keywords;  # key words of tests to skip
 my %ignored_keywords;   # key words of tests to ignore results
 my %enabled_keywords;   # key words of tests to run
+my %disabled_features;  # features of tests to skip
+my %ignored_features;   # features of tests to ignore results
```

Comment:
I'm not sure the value of ignoring based on features. When do you anticipate this being useful?

Reply:
TBH, I only added `ignored` option to have a more "complete" implementation like `keyword`. In this case, I still don’t see any scenario where it could be useful.

Reply:
First of all thanks for the review.

TBH, I only added `ignored` option to have a more "complete" implementation like `keyword`. In this case, I still don’t see any scenario where it could be useful.